README.md

```diff
@@ -110,62 +64,48 @@ sudo rm -rf **\~/.solcx**
 sudo rm -rf **\~/.vvm**
 ```
 
-## Environment Variables:
+### Quick Start 
 
-Some plugins require environment variables to connect to their external systems, such project IDs or API keys.
-Follow instructions from individual plugin documentations, such as:
+After you installed Ape, there are 2 ways to interact with Ape:
+- Terminal Interaction
+- Python Reference
 
-* [ape-alchemy](https://github.com/ApeWorX/ape-alchemy/blob/main/README.md#quick-usage)
-* [ape-infura](https://github.com/ApeWorX/ape-infura#readme)
-
-Generally, set environment variables by doing the following:
+Terminal interaction is a way for you to open a project and manually 
+you will want to check to make sure it works and you can see the list of commands:
 
 ```bash
-# Used by the `ape-infura` plugin
-export WEB3_INFURA_PROJECT_ID=<YOUR_INFURA_PROJECT_ID>
-# Used by the `ape-alchemy` plugin
-export WEB3_ALCHEMY_API_KEY=<YOUR_ALCHEMY_KEY>
-```
-
-Place these in environment files, such as your `.bashrc` or `.zshrc`.
+$ ape
+Commands:
```

Comment:
Risky putting this here because if we add another command or even change the short-help text of a command, we may forgot to update this part in the README

Reply:
I have rm in favour of `ape --version`